contracts/CO30_Matias_MiseryBusiness.porto/initPlayerLocal.sqf

```diff
@@ -0,0 +1,6 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[topsecret, 1, true, 1, "Photograph and secure document", [], 5, "Top Secret Document", "This document details the dealings of the company with CSAT higher up."] call zen_modules_fnc_addIntelAction;
+[docs, 1, true, 1, "Photograph and secure document", [], 5, "Bank statements", "This document contains bank statements from the company, which show transfers to and from CSAT accounts."] call zen_modules_fnc_addIntelAction;
+[secretdoc, 1, true, 1, "Photograph and secure document", [], 5, "Transport log", "This document contains a detailed record of every piece of cargo that has come and gone from the island."] call zen_modules_fnc_addIntelAction;
```

Comment:
You should use physical items for this, rather than items that will disappear when a person picks them up. This interaction should only be used for reading actions, not taking.

Or change the `true` to false to prevent the items from deleting, and just being read.

Reply:
The way I thought of it was that they'd photograph and secure the doc. That's why I chose to delete it, cause they're being secured

Reply:
The problem is that it no logner exists, they "secure" it, but then it is deleted. What if the person that secures it dies? There is no way to get it off of their body, cause it doesn't actually exist.

Reply:
Forgot I'd done this, but it's set to show for the entire group of the person who found it, as if they'd taken pictures sent them. It just doesn't simulate the person having the item in the inventroy, but it does give everyone the intel.